sampleSpecs/defaults.smithy

```diff
@@ -1,66 +1,74 @@
 $version: "2.0"
 
 metadata smithy4sDefaultRenderMode = "FULL"
-
 namespace smithy4s.example
 
 list StringList {
-  member: String
+    member: String
 }
 
 map DefaultStringMap {
-  key: String
-  value: String
+    key: String
+    value: String
 }
 
 structure DefaultTest {
-  one: Integer = 1
-  two: String = "test"
-  three: StringList = []
-  @default
-  four: StringList
-  @default
-  five: String
-  @default
-  six: Integer
-  @default
-  seven: Document
-  @default
-  eight: DefaultStringMap
-  @default
-  nine: Short
-  @default
-  ten: Double
-  @default
-  eleven: Float
-  @default
-  twelve: Long
-  @default
-  thirteen: Timestamp
-  @default
-  @timestampFormat("http-date")
-  fourteen: Timestamp
-  @default
-  @timestampFormat("date-time")
-  fifteen: Timestamp
-  @default
-  sixteen: Byte
-  @default
-  seventeen: Blob
-  @default
-  eighteen: Boolean
+    one: Integer = 1
+    two: String = "test"
+    three: StringList = []
+    @default
+    four: StringList
+    @default
+    five: String
+    @default
+    six: Integer
+    @default
+    seven: Document
+    @default
+    eight: DefaultStringMap
+    @default
+    nine: Short
+    @default
+    ten: Double
+    @default
+    eleven: Float
+    @default
+    twelve: Long
+    @default
+    thirteen: Timestamp
+    @default
+    @timestampFormat("http-date")
+    fourteen: Timestamp
+    @default
+    @timestampFormat("date-time")
+    fifteen: Timestamp
+    @default
+    sixteen: Byte
+    @default
+    seventeen: Blob
+    @default
+    eighteen: Boolean
 }
 
 structure DefaultOrderingTest {
-  one: Integer = 1
-  two: String
-  @required
-  three: String
+    one: Integer = 1
+    two: String
+    @required
+    three: String
 }
 
 @mixin()
 structure DefaultInMixinTest {
-  one: String = "test"
+    one: String = "test"
+}
+
+structure DefaultVariants {
```

Comment:
ahh, until I saw this structure I was unsure what use case we were not covering properly but now it's clearer

a field *not marked with required* with a default would use `required` before, now it use `field`